docs/developer/index.rst

```diff
@@ -13,14 +13,16 @@ Before you submit a pull request to Bodhi, please ensure that it meets these cri
 * All tests must pass.
 * New code must have 100% test coverage. This one is particularly important, as we don't want to
   deploy any broken code into production. At the end of ``btest`` run, you can see your code coverage.
+  If you are not using Vagrant environment you can check the code coverage by running
+  ``diff-cover coverage.xml --compare-branch=origin/develop --fail-under=100``.
 * New functions, methods, and classes must have docblocks that explain what the code block is, and
   describing any parameters it accepts and what it returns (if anything). You can use the
-  ``pydocstyle`` utility to automatically check your code for this. You can also run ``bci pydocstyle``
+  ``pydocstyle`` utility to automatically check your code for this. You can also run ``bteststyle``
   in Vagrant.
 * Parameter and return value types should be declared using `type hints`_. You can test this by running
-  ``bci mypy`` in Vagrant.
+  ``bteststyle`` in Vagrant.
```

Comment:
This can stay, running `bci mypy` is currently the only way to run *only* mypy in Vagrant